templates/provider/hmc/templates/rbac/user-facing/clustertemplatechains-creator.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ include "hmc.fullname" . }}-clustertemplatechains-creator-role
+rules:
+  - apiGroups:
+      - hmc.mirantis.com
+    resources:
+      - clustertemplatechains
+    verbs: {{ include "rbac.viewerVerbs" . | nindent 6 }}
```

Comment:
Is this an error? 

It looks like it should have rights to create those. Now it has the same permissions as `clustertemplatechains-viewer-role`

Reply:
it's viewerVerbs + create (the next line)